cfpb/cfgov-refresh
# Submit a Complaint: Add ask search organism

## Additions

- Adds Ask Search organism to Wagtail.

## Changes

- Migrates Ask Search to organism structure from Ask structure.

## Testing

1. Pull branch and run `python cfgov/manage.py migrate`
2. Log into local Wagtail.
3. Create a new sublanding page.
3. Add a "Full width text" 
4. Add a "Well with ask search"
5. Fill in the options and Preview the page.
6. Ask CFPB search (/ask-cfpb/) should work as before (only change should be markup has gone from `<aside>` to `<div>` (sorry)).

## Screenshots

<img width="1231" alt="Screen Shot 2019-09-11 at 6 27 00 PM" src="https://user-images.githubusercontent.com/704760/64740002-e22eb800-d4c1-11e9-829a-e06f4dcb4e03.png">

## Notes
- This only adds the Well With Ask Search to the Full Text Width organism, since that's where it's needed for the Submit a Complaint speedbump (see https://[GHE]/CFGOV/platform/issues/3112), but it could be added to the sublanding page too, like the regular well is. I figure it's better to add it where it's needed now only though.

## Todos

- I need to add the magnifying glass icon to the search input. But I may do that in another PR.
